### PR TITLE
Added i18n to register

### PR DESCRIPTION
**Pull request by rlajous**
## Summary 

- Added translation to Register view

## Trello Card

https://trello.com/c/eRRvXM4i/17-internationalization

### CODE REVIEW

File: wolox-books/src/router.js [hunk 1]
```diff
@@ -17,7 +17,8 @@ export default new Router({
       // route level code-splitting
       // this generates a separate chunk (about.[hash].js) for this route
       // which is lazy-loaded when the route is visited.
-      component: () => import(/* webpackChunkName: "about" */ './views/About.vue')
+      component: () =>
+        import(/* webpackChunkName: "about" */ './views/About/index.vue')
```
Comment (mcallegari10) [inline wolox-books/src/router.js]: Write this in the same line

File: wolox-books/src/router.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import Vue from 'vue'
 import Router from 'vue-router'
-import Home from './views/Home.vue'
+import Home from './views/Home/index.vue'
```
Comment (mcallegari10) [inline wolox-books/src/router.js]: Remember to delete the folder and just stay with the component

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: wolox-books/src/router.js (hunk 1 above)
Comment (rlajous) [reply wolox-books/src/router.js]: eslint autofixing this

File: wolox-books/src/router.js (hunk 2 above)
Comment (rlajous) [reply wolox-books/src/router.js]: this is fixed in the other pr

File: wolox-books/src/main.js [hunk 3]
```diff
@@ -1,14 +1,22 @@
...
+import { Register as messages } from "./config/i18n";
 
-Vue.use(Vuelidate)
-Vue.config.productionTip = false
+Vue.use(VueI18n);
+
+const i18n = new VueI18n({
+  locale: "en",
+  messages // set locale messages
+});
```
Comment (FrankIglesias) [inline wolox-books/src/main.js]: remove this semi colon

File: wolox-books/src/main.js [hunk 4]
```diff
@@ -1,14 +1,22 @@
-import Vue from 'vue'
-import App from './App.vue'
-import router from './router'
-import store from './store'
-import Vuelidate from 'vuelidate'
+import Vue from "vue";
```
Comment (FrankIglesias) [inline wolox-books/src/main.js]: remove semi colon

File: wolox-books/src/main.js [hunk 5]
```diff
@@ -1,14 +1,22 @@
...
-Vue.use(Vuelidate)
-Vue.config.productionTip = false
+Vue.use(VueI18n);
+
+const i18n = new VueI18n({
+  locale: "en",
+  messages // set locale messages
+});
+
+Vue.config.productionTip = false;
```
Comment (FrankIglesias) [inline wolox-books/src/main.js]: same (why this false?

File: wolox-books/src/router.js [hunk 6]
```diff
@@ -1,23 +1,15 @@
...
-Vue.use(Router)
+Vue.use(Router);
 
 export default new Router({
   routes: [
     {
-      path: '/',
-      name: 'home',
+      path: "/",
+      name: "home",
```
Comment (FrankIglesias) [inline wolox-books/src/router.js]: check your linter

File: wolox-books/src/views/Register.vue [hunk 7]
```diff
@@ -1,22 +1,21 @@
...
-    label.label(for="password")
-      | Password
-    input#password(type="password" name="password" v-model="password" v-model.trim="$v.password.$model")
-    button
-      | Sign Up
-  a(href="/")
-    | Log In
+      | {{$t('register.lastName')}}
+    input#lastName(type="text",name="lastName",v-model="lastName")
+    label.label(for="email")
```
Comment (FrankIglesias) [inline wolox-books/src/views/Register.vue]: remove this extra space

File: wolox-books/src/views/Register.vue [hunk 8]
```diff
@@ -1,22 +1,21 @@
...
-    | Log In
+      | {{$t('register.lastName')}}
+    input#lastName(type="text",name="lastName",v-model="lastName")
+    label.label(for="email") 
+      | {{$t('register.email')}}
+    input#email(type="text",name="email",v-model="email")
+    label.label(for="password") 
+      | {{$t('register.password')}}
+    input#password(type="password",name="password",v-model="password")
+    button
```
Comment (FrankIglesias) [inline wolox-books/src/views/Register.vue]: same

File: wolox-books/src/views/Register.vue [hunk 9]
```diff
@@ -1,22 +1,21 @@
...
+    input#lastName(type="text",name="lastName",v-model="lastName")
+    label.label(for="email") 
+      | {{$t('register.email')}}
+    input#email(type="text",name="email",v-model="email")
+    label.label(for="password") 
+      | {{$t('register.password')}}
+    input#password(type="password",name="password",v-model="password")
+    button 
+      | {{$t('register.signUp')}}
+  a(href="/")
```
Comment (FrankIglesias) [inline wolox-books/src/views/Register.vue]: same

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: wolox-books/src/main.js (hunk 5 above)
Comment (rlajous) [reply wolox-books/src/main.js]: it came as default like that

File: wolox-books/src/router.js [hunk 10]
```diff
@@ -1,6 +1,6 @@
 import Vue from 'vue'
 import Router from 'vue-router'
-import Home from './views/Home.vue'
+import Home from './views/Register'
```
Comment (mcallegari10) [inline wolox-books/src/router.js]: The name of the import should be the same you are importing, `Register` in this case

File: wolox-books/src/router.js [hunk 11]
```diff
@@ -9,12 +9,7 @@ export default new Router({
     {
       path: '/',
       name: 'home',
-      component: Home
-    },
-    {
-      path: '/about',
-      name: 'about',
-      component: () => import(/* webpackChunkName: "about" */ './views/About.vue')
+      component: Register
```
Comment (mcallegari10) [inline wolox-books/src/router.js]: Maintain the dynamic import `component: () => import(/* webpackChunkName: "about" */ './views/Register.vue')`, that way performance will improve :smile:

File: wolox-books/src/views/Register.vue [hunk 12]
```diff
@@ -1,22 +1,22 @@
 <template lang='pug'>
 .home
-  form.register-form(@submit.prevent='onSubmit' novalidate)
+  form.register-form(@submit.prevent='onSubmit')
     label.label(for='firstName')
-     | First Name
+      |{{$t('register.firstName')}}
```
Comment (mcallegari10) [inline wolox-books/src/views/Register.vue]: Leave a space between pipe and text

**Review by FrankIglesias — APPROVED** [review]

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**